app/src/main/java/com/casadelfuego/omnikiller/ocr/CreditCardValidator.kt

```diff
@@ -0,0 +1,99 @@
+package com.casadelfuego.omnikiller.ocr
+
+object CreditCardValidator {
```

Comment:
I don't see any reason to make this a singleton,
I prefer keeping this a "normal" class, then inject in the classes we need it. Then, it will easier to mock this validator when testing the classes that uses this

Reply:
Porque ta comentando em ingles?